scripts/jestPuppeteer.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  launch: {
```

Comment:
This config also takes a `server` option that you can use to have Jest run the dev server. That might mean you can remove `run_integration_tests.sh`

Reply:
`./run_integration_tests` has a timeout for the server to start - perhaps you could just use that script without the bit which actually runs the tests?